www/templates/default/html/js/manager.js

```diff
@@ -1,3 +1,7 @@
+require([frontend_url + "templates/default/html/js/vendor/select2/js/select2.min.js"], function(select2) {
```

Comment:
fixed in https://github.com/mfairchild365/UNL_UCBCN_System/commit/6c009d65d3fb6d6965decb8984c9079f40eb8655